docs/cvmanager.md

```diff
@@ -0,0 +1,142 @@
+# Migrating from `cvmanager` to Foreman Ansible Modules
+
+[cvmanager](https://github.com/RedHatSatellite/katello-cvmanager) is a tool to automate Content View management workflows. It hasn't been updated since a long time and users are encouraged to migrate to Ansible and the Foreman Ansible Modules.
+
+This document documents how the various workflows can be translated into Ansible playbooks.
+
+Please note that `cvmanager` configuration refers to Content Views by their labels, while Ansible uses the names of the Content Views for that.
+
+## Cleanup of old Content Views
+
+[TBD](https://github.com/theforeman/foreman-ansible-modules/pull/958)
```

Comment:
I don't think this should block this PR.

Reply:
> I don't think this should block this PR.

Agreed.